lib/streamlit/elements/dataframe_selector.py

```diff
@@ -34,15 +34,16 @@ def dataframe(self, data=None, width=None, height=None):
 
         Parameters
         ----------
-        data : pandas.DataFrame, pandas.Styler, numpy.ndarray, Iterable, dict,
-            or None
+        data : pandas.DataFrame, pandas.Styler, numpy.ndarray, Iterable, dict, or None
             The data to display.
 
             If 'data' is a pandas.Styler, it will be used to style its
             underyling DataFrame. Streamlit supports custom cell
             values and colors. (It does not support some of the more exotic
             pandas styling features, like bar charts, hovering, and captions.)
             Styler support is experimental!
+            If 'dataFrameSerialization' config option is set to 'arrow',
+            'data' can also be of type pyarrow.Table.
```

Comment:
Is this supposed to be _must_? Meaning, if you set `dataFrameSerialization = 'arrow'`, but don't provide a `pyarrow.Table`, does it still work?

Secondarily, do we use camel case elsewhere in the public api? Couldn't we call that argument `serialization` to avoid the multiple word nature?

Reply:
Re: "should this be _must_" - no, this is an _additional_ feature you get when using Arrow. ("Legacy dataframes" don't support `pyarrow.Table`, "Arrow dataframes" support everything that legacy does, plus `pyarrow.Table`)

Re: "do we use camel case" - yeah, our multi-word config options are all camel-cased (e.g. "server.enableWebsocketCompression").